### PR TITLE
naming convention changes and hostname updates 

### PR DESCRIPTION
**Pull request by pyang55**

### CODE REVIEW

**Comment by pyang55** [comment]
@catsby pull request here

**Comment by woz5999** [comment]
lgtn

File: logicmonitor/config.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ func (c *Config) NewLMClient() (*lmv1.DefaultApi, error) {
 		},
 	}
 	config.BasePath = "https://" + c.Company + ".logicmonitor.com/santaba/rest"
+	config.UserAgent = "terraform-provider-logicmonitor"
```
Comment (andrewrynhard) [inline logicmonitor/config.go]: Can we do something like
```
"LogicMonitor Terraform Provider"
```

@woz5999 Opinion?

**Comment by catsby** [comment]
😄

**Pull request merged**